src/css/in-page.css

```diff
@@ -0,0 +1,347 @@
+  width: 26px;
+}
+
+.fx-relay-menu-logo-text {
+  width: 94px;
+}
+
+.fx-relay-menu-dashboard-link {
+  padding: var(--spacingXs);
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  position: relative;
+}
+
+.fx-relay-menu-dashboard-link:hover,
+.fx-relay-menu-dashboard-link:focus {
+  background-color: var(--colorGrey10);
+  border-radius: 100%;
+}
+
+.fx-relay-menu-dashboard-link span {
+  display: none;
+  color: var(--colorWhite);
+  background-color: var(--colorGrey40);
+  border-radius: var(--borderRadiusSm);
+  padding: var(--spacingXs) var(--spacingSm);
+  position: absolute;
+  top: calc(100% + var(--spacingXs));
+  right: 0;
+}
+
+.fx-relay-menu-dashboard-link:hover span,
+.fx-relay-menu-dashboard-link:focus span {
+  display: block;
+  margin: 0 var(--spacingXs);
+}
+
+/* Main Content */
+.fx-relay-menu {
+  padding: 0;
+  position: relative;
+}
+
+.fx-relay-menu-content {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: var(--spacingMd) var(--spacingMd) 0;
+}
+
+.fx-relay-menu-content.fx-content-signed-in-premium {
+  padding: 0;
+}
+
+.fx-relay-menu-content.is-hidden {
+  display: none;
+}
+
+.fx-relay-menu-content p {
+  margin: 0 0 var(--spacingMd);
+  padding: 0;
+  text-align: center;
+}
+
+/* Mask List */
+
+.fx-relay-menu-masks-lists {
+  margin-top: var(--spacingSm);
+}
+
+.fx-relay-menu-masks-list {
+  border-radius: var(--borderRadiusSm);
+  background-color: var(--colorGrey10);
+  margin-bottom: var(--spacingMd);
+  overflow: hidden;
+}
+
+.fx-relay-menu-masks-list.t-website {
+  border-radius: var(--borderRadiusSm);
+  background-color: var(--colorWhite);
+  outline: 2px solid var(--colorBlue50);
+}
+
+.fx-relay-menu-masks-list.t-website span {
+  border-bottom: 1px solid var(--colorGrey10);
+}
+
+.fx-relay-menu-masks-list.t-website button {
+  color: var(--colorBlue50);
+}
+
+.fx-relay-menu-masks-list.t-search {
+  border-radius: 0;
+  background-color: transparent;
+  outline: 0;
+  margin-bottom: 0;
+}
+
+.fx-relay-menu-masks-list.t-search ul li {
+  border-bottom: 1px solid var(--colorGrey10);
+}
+
+.fx-relay-menu-masks-list.t-search ul li:last-child {
+  border-bottom: none;
+}
+
+.fx-relay-menu-masks-list.t-search button {
+  /* color: var(--colorBlue50); */
+}
+
+.fx-relay-menu-masks-list > span {
+  border-bottom: 1px solid var(--colorGrey20);
+  padding: var(--spacingSm);
+  font-size: 12px;
+  line-height: 1rem;
+  text-align: center;
+  display: block;
+  color: var(--colorGrey50);
+}
+
+.fx-relay-menu-masks-list ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.fx-relay-menu-masks-list li {
+  display: block;
+  width: 100%;
+}
+
+.fx-relay-menu-masks-list li.is-hidden {
+  display: none;
+}
+
+.fx-relay-menu-masks-list button {
+  appearance: none;
+  display: block;
+  outline: none;
+  border: none;
+  background-color: transparent;
+  width: 100%;
+  padding: var(--spacingSm) var(--spacingMd);
+  cursor: pointer;
+  text-align: left;
```

Comment:
Great catch. I fixed this instance and another one over in `popup.css`

Note: This would be a great linter rule for https://github.com/mozilla/fx-private-relay-add-on/pull/316

Revised in 05460091ed852a38c39cfd8650d0daebe466c80f

Reply:
It would be, great suggestion! I just tried to add it, but unfortunately I couldn't find a stylelint rule that would detect it :(